### DISCUSSION TITLE
Deployment without Kubernetes

### DISCUSSION THREAD

**Discussion by sycured** [fission/fission#2500] [Q&A] [answered]
Hello,

Is it possible to deploy fission without using Kubernetes?
Are you using specific Kubernetes APIs?

In my case, I've a [Nomad](https://www.nomadproject.io/) cluster

Thank you

**Comment by sanketsudake** [comment, marked as answer]
Hi Currently its not possible to have Fission running non kubernetes cluster, as fission is tightly coupled with Kubernetes APIs and custom resources ability.
